draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -257,7 +257,7 @@ in Section 7.2.1. It is thus implicitly clear that the quoted rule only applies
 
 With the exception of pacing, this specification conservatively limits the growth in cwnd, similar to Cubic and SCTP.
 The exception for pacing in the second paragraph requires that when pacing is enabled, it is specifically taken into account. Pacing could occur over various timescales, but
-is typically done with delays below an RTT; thus, rule #2 in {{rules}} should cover this case.
+is typically done with delays below an RTT; thus, rule #2 in {{rules}} covers this case, and it removes the unnecessary pacing limitation.
```

Comment:
What about instead saying that RFC 9002 is consistent with this draft, and no specific update is needed?  I don't think of the second paragraph in Section 7.8 as an unnecessary limitation; I view it as an implementation clarification in the same way that the distinction between FlightSize and cwnd may need to be clearly understood in implementations.

Reply:
Ahh, I see, this is quite unclear text from me. I agree that the unnecessary pacing limitation is not related to the second paragraph of Section 7.8 of RFC 9002, even though it certainly reads that way.  I do think we need to update RFC 9002 because, like the other specs we update, it's unnecessarily strict - it doesn't allow cwnd to grow based on maxFS as we do.

I'll propose an update.